ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -198,6 +202,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: node.started ? 'started' : 'stopped',
       version: node.pkg.version,
       git: node.pkg.git,
+      nodeName: node.config.config.nodeName,
```

Comment:
`config.config` is not meant to be used. Use config.ge() instead like this:

```js
config.get('nodeName')
```

Reply:
thanks, updated